src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    ArrayList<Task> tasks;
+    public int items;
+
+    public TaskList(ArrayList<Task> t, int number) {
```

Comment:
Perhaps a more descriptive variable name here? For instance, task and items, which corresponds to your overall implementation of this class! I noticed the variables in the other methods in this class may be changed in a similar way as well.